src/components/CheckBox/CheckboxList.test.tsx

```
@@ -0,0 +1,534 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
```

ESLint

[error] 'userEvent' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)